.github/workflows/daphneci.yml

```
@@ -30,3 +30,34 @@ jobs:
         run: cargo build --release
       - name: Testing
         run: cargo test -- --nocapture
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v3
+      - name: 🏷️ Docker meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            cloudflare/daphne-worker-helper
+          flavor: |
+            latest=auto
+          tags: |
+            type=semver,pattern={{version}}
+            type=sha
+      - name: 👷 Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: 🔏 Login to Docker Hub
+        uses: docker/login-action@v2
+        if: github.event_name != 'pull_request'
```

Would it make sense to explicitly match on the expected event in case `pull_request_review` or other events are added later on?
```suggestion
        if: github.event_name == 'push'
```

---

It would be best. I've updated the commit accordingly.
I've also updated the actions to be referenced by sha instead of by tag.

---

LGTM. I would not be too worried about using a tag instead of digest, but it does not hurt I suppose. If the Docker or GitHub organization cannot even maintain integrity of their own images, then they have bigger issues to worry about.